test/resolve-config.test.js

```diff
@@ -17,7 +17,7 @@ test('Returns user config', t => {
       gitlabApiUrl: urlJoin(gitlabUrl, gitlabApiPathPrefix),
       assets,
       milestones: undefined,
-      postComments,
+      successComment: undefined,
```

Comment:
do these still need to be explicitly set to `undefined` here?

Reply:
Yes, because the property will be there, even if not passed via properties, and the tests do a deep equality comparison.